versioned_docs/version-23.10/installation/technical.md

```diff
@@ -263,18 +263,18 @@ Many components are used to build a central Centreon server:
 
 * Apache web server for Centreon web interface
 * MariaDB databases to store Centreon configuration parameters as well as monitoring and performance data
-* The Centreon Gorgone process is used to send monitoring configuration to the remote server and to manage it
+* The Centreon Gorgone process is used to send monitoring configuration parameters to the remote server and to manage it
```

Comment:
the monitoring configuration? Here the word configuration refers to something specific, we need to discuss it.